### DISCUSSION TITLE
Overlapping 'scroll back to top' element

### DISCUSSION THREAD

**Discussion by krishna-santosh** [nunocoracao/blowfish#285] [General]
Hello @nunocoracao , Thank you for creating this beautiful theme 😄

Found a small issue with the **scroll back to top** element, It is overlapping with the footer text.

![Screenshot_20221206_164517-1](https://user-images.githubusercontent.com/75202541/205897214-771fb67a-d037-4a71-97fb-e07dc4207f1c.png)

![Screenshot_20221206_163058-1](https://user-images.githubusercontent.com/75202541/205895992-54def211-4087-45ed-9521-d552b2ba96ff.png)

If this behavior is already intended, that's cool, if not what are your thoughts on this?

**Comment by nunocoracao** [comment]
hello @krishna-santosh that's a nice one... we need to work on it... Will create an issue from this discussion. If you want to submit a PR feel free to do so.

**Reply by krishna-santosh** [reply]
Thanks @nunocoracao , Submitted a PR #296 
![Screenshot_20221211_185829-1](https://user-images.githubusercontent.com/75202541/206906594-871d448e-9dbc-4759-bd30-5c30d0ddd12a.png)
![Screenshot_20221211_185629-1](https://user-images.githubusercontent.com/75202541/206906669-b7e46c95-bc98-4bee-aa7d-31c614f75aec.png)

**Reply by nunocoracao** [reply]
Thanks @krishna-santosh I'll review and add to the next release
